src/main/resources/templates/circleProfile.html

```diff
@@ -116,6 +122,23 @@ <h4 th:text="*{name}">Full Name</h4>
     <span class="list-end" id="no-results"></span>
 
     <div th:replace="MainLayout :: footer"></div>
+    <script>
+        function increaseInterest() {
+            return fetch("/api/provider/(Burgir)/increaseInterest", { // TODO hook in provider?
```

Comment:
I might use sg. like:

```
function increaseInterest(provider) {
    return fetch(`/api/provider/${provider}/increaseInterest`, {
```

Reply:
This gets really ugly (alias is injected as JavaScript from a template, and I do not like the sound of it :/ ). Or would it be okay? Or you had something else in mind?

For reference, this is how I understood your comment:
```patch
diff --git a/src/main/resources/templates/circleProfile.html b/src/main/resources/templates/circleProfile.html
index 7bc09ae..af6b985 100644
--- a/src/main/resources/templates/circleProfile.html
+++ b/src/main/resources/templates/circleProfile.html
@@ -58,7 +58,7 @@
                         <tr>
                             <td th:text="#{lang.profile.open}">Please open!</td>
                             <td>
-                                <button id="interest-counter" class="action-button" onclick="increaseInterest(1); return false" th:text="${selectedCircle?.interestCounter}" class="action-button">0</button>
+                                <button id="interest-counter" class="action-button" th:onclick="increaseInterest(${selectedCircle.alias}); return false" th:text="${selectedCircle?.interestCounter}" class="action-button">0</button>
                             </td>
                         </tr>
                         <tr>
@@ -123,8 +123,8 @@

     <div th:replace="MainLayout :: footer"></div>
     <script>
-        function increaseInterest() {
-            return fetch("/api/provider/(Burgir)/increaseInterest", { // TODO hook in provider?
+        function increaseInterest(provider) {
+            return fetch(`/api/provider/${provider}/increaseInterest`, {
                 method: 'POST',
                 cache: 'no-cache',
                 credentials: 'same-origin',
```

Reply:
In a normal scenario you could write sg. like:
```
th:onclick="|increaseInterest(${selectedCircle.alias}); return false|"
```
Notice the string template literal vertical bars.

But in case of th:onclick you are not able to use variables, so you have two options:
- use data atibutes: https://stackoverflow.com/a/54840163
- use inlined javascript template:
```
<script th:inline="javascript">
const provider = /*[[${selectedCircle.alias}]]*/ '';
</script>
```

Reply:
I forgot to reflect the change in the comment.

Reply:
I have used data attributes in the changes, can you check if it works?

Reply:
for you :D